domains/mohammedobaid.json

```
@@ -0,0 +1,10 @@
+{
+    "description": "My name is mohammed obaid. I am a software engineer",
+    "owner": {
+        "username": "mdobaid311",
+        "email": "mdobaid311@gmail.com"
+    },
+    "record": {
+        "CNAME": "http://mohammedobaid.vercel.app/"
```

```suggestion
        "CNAME": "mohammedobaid.vercel.app"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).